promscale/installation/recomm-guide.md

```diff
@@ -0,0 +1,40 @@
+# Resource recommendation guide
+
+The resource recommendation guide is to help you understand the compute and disk requirements for Promscale based on your ingest rate and retention. 
+
+## Metrics
```

Comment:
Don't leave a heading with no content. Can we add a sentence or two here to introduce the sub-sections?